test/relations.test.js

```diff
@@ -873,8 +928,8 @@ describe('relations', function() {
       Physician.create(function(err, physician) {
         physician.patients.create({name: 'a'}, function(err, ch) {
           id = ch.id;
-          physician.patients.create({name: 'z'}, function() {
-            physician.patients.create({name: 'c'}, function() {
+          physician.patients.create({name: 'z'}, function(err, ch) {
+            physician.patients.create({name: 'c'}, function(err, ch) {
```

Comment:
What's the purpose for this change?

I am ok to keep it in if you also add error handling.

```js
physician.patients.create({name: 'z'}, function(err, ch) {
  if (err) return done(err);
  physician.patients.create({name: 'c'}, function(err, ch) {
    if (err) return done(err);
    verify(physician);
  });
});
```